test/interface/test_cli_with_args.py

```diff
@@ -1,13 +1,25 @@
 import argparse
 
 import pytest
+
+from src.client import _Client
 from src.interface import ArgInterface
-from src.tables import CTran_Data
 
 
 # @pytest.fixture
 # def ctran():
 #     return CTran_Data(user="sw23", passwd="fake", hostname="localhost", db_name="aperture", schema="aperture", verbose=True)
+#
+#
+# def test_db(ctran):
+#     ai = ArgInterface()
+#     df = ai.query_with_args(ctran, ['--date-start=2019-03-01', '--date-end=2019-03-01'])
+#     assert len(df.index) == 71084
+
+@pytest.fixture
+def cl():
+    return _Client()
```

Comment:
You may want to see test_client for how I mocked out a client instance - client will automatically use config.load() which may be undesirable.

Reply:
Ah, thank you, I will review that and change this in the near-future.